multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -331,11 +360,11 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	}
 
 	if !epNoChange {
-		klog.InfoS("Endpoints have new change, update ResourceExport", "endpoints",
+		klog.InfoS("Endpoints or EndpointSlices have new change, update ResourceExport", "service",
 			req.String(), "resourceexport", epExportNSName)
 		err = r.endpointsHandler(ctx, req, eps, epResExportName, re, r.remoteCommonArea)
```

Comment:
EndpointSlice is introduced to reduce update size when only one slice changes. If we always merge all EndpointSlices, we lose the benefit. As I do not really know any real user of Endpoints export, I am fine with your current approach if it is simpler than exporting individual EndpointSlice, but at least add some comments to explain the current behavior.

Reply:
Sure, added a few comments before `getSubsetsFromEndpointSlice`.